store/tikv/region_cache_test.go

```diff
@@ -143,33 +195,6 @@ func (s *testRegionCacheSuite) TestUpdateLeader(c *C) {
 	c.Assert(s.getAddr(c, []byte("z")), Equals, s.storeAddr(s.store2))
 }
 
-func (s *testRegionCacheSuite) TestUpdateLeader2(c *C) {
-	loc, err := s.cache.LocateKey(s.bo, []byte("a"))
-	c.Assert(err, IsNil)
-	// new store3 becomes leader
-	store3 := s.cluster.AllocID()
-	peer3 := s.cluster.AllocID()
-	s.cluster.AddStore(store3, s.storeAddr(store3))
-	s.cluster.AddPeer(s.region1, store3, peer3)
-	// tikv-server reports `NotLeader`
```

Comment:
Hi @disksing, have a question, when kv add a new store3 and became leader, it seems kv should return EpochNotMatch instead of NotLeader?